draft-ietf-webpush-protocol.xml

```diff
@@ -217,6 +216,13 @@
           </list>
         </t>
         <t>
+          Examples do not demonstrate the use of HTTP alternative services (<xref target="connecting"/>),
+          which would minimally impact the example exchanges between the User Agent and the Push Service.
+          The one minor, visible change would be the addition of the Alt-Used header (<xref target="RFC7838"/>)
+          in requests from the User Agent to the Push Service. For simplicity, this header has been
+          omitted from examples.
+        </t>

```

Comment:
I think that you missed here.  Maybe:

> Examples show use of HTTPS over port 443, rather than the registered port.  A push service deployment might choose to do this to maximize chances that user agents are able to reach the service.  If a push service uses <xref target="RFC7838">Alternative Services</xref> to direct user agents to port WEBPUSH-PORT, as they might do to gain the benefits of the standardized port without sacrificing reachability (see <xref target="connecting"/), this would only be apparent in examples through the inclusion of an Alt-Used header field.